.gitignore

```diff
@@ -1 +1,2 @@
 scripts/output
+scripts/config.json
```

Comment:
Is this a good change? Not sure about others but I know that I would want my custom `config.json` to be in source control.

Reply:
My view is it's a good change for our repo as we intentionally don't store any config beyond the example, you may wish to override it for your local repos

@nightah you use this regularly as well, thoughts?

Reply:
It's to ensure that a user doesn't accidently commit their configuration file back to the main repo.
If you want to maintain your own repo and commit to said repo you can do that even with that entry in the `.gitignore`.

Reply:
@DragonQ fyi `git add config.json --force` will override the gitignore